01/app.js

```diff
@@ -1 +1,9 @@
-console.log('DOM');
\ No newline at end of file
+console.log("DOM");
+
+const pElement = document.querySelector(".comments__item--newest");
```

Comment:
Treść zadania mówiła, że element ma mieć 2 klasy, u Ciebie ma tylko jedną.
Może to też działa, ale nie jest zgodne z założeniami. Powinno być: `document.querySelector(".comments__item.comments__item--newest");`